plugins/obs-ffmpeg/obs-ffmpeg-srt.h

```diff
@@ -750,9 +750,15 @@ static int libsrt_open(URLContext *h, const char *uri)
 	if (ret < 0)
 		goto err;
 
+#ifdef _WIN32
 	struct timeb timebuffer;
 	ftime(&timebuffer);
 	s->time = (double)timebuffer.time + 0.001 * (double)timebuffer.millitm;
+#else
+	struct timespec timesp;
+	clock_gettime(CLOCK_REALTIME, &timesp);
+	s->time = (double)timesp.tv_sec + 0.000000001 * (double)timesp.tv_nsec;
```

Comment:
We should probably use the `ulti_mul_div64` helper an then cast the result to a double to be consistent with audio handling and hopefully avoid any accuracy issues.

Reply:
Sorry I was thinking of a different operation. we dont need the extra mul from that util.